install/kubernetes/cilium/values.yaml

```diff
@@ -470,14 +474,41 @@ hubble:
       # false, the certs for Hubble server need to be provided by setting
       # appropriate values below.
       enabled: true
-      # Sets the method to auto-generate certificates. Currently, only "helm"
-      # is supported. This method uses Helm to generate certificates.
+      # Sets the method to auto-generate certificates. Supported values:
+      # - helm:      This method uses Helm to generate all certificates.
+      # - cronJob:   This method uses a Kubernetes CronJob the generate any
+      #              certificates not provided by the user at installation
+      #              time.
       method: helm
       # Generated certificates validity duration in days.
       certValidityDuration: 1095
-    # base64 encoded PEM values for the Hubble CA certificate
+      # If hubble.tls.auto.method=cronJob, these values are used
+      # for the Kubernetes CronJob which will be scheduled regularly to
+      # (re)generate any certificates not provided manually.
+      cronJob:
+        # certgen image used in the job pod spec
+        image:
+          repository: docker.io/cilium/certgen
+          tag: v0.1.0
```

Comment:
Since this is a CronJob I would suggest to set the sha256 part of the tag to avoid accidentally pushes to the `v0.1.0` and people assuming the `v0.1.0` is still safe.